packages/api/swagger-config.json

```diff
@@ -1,171 +1,183 @@
+            "operationId": "post-name",
+            "requestBody": {
+               "required": true,
+               "content": {
+                  "text/plain": {
+                     "schema": {
+                        "type": "string"
+                     }
+                  },
+                  "application/octet-stream": {
+                     "schema": {
+                        "type": "string",
+                        "format": "binary"
+                     }
+                  }
+               }
+            },
+            "parameters": [
+               {
+                  "name": "key",
+                  "in": "path",
+                  "required": true,
+                  "schema": {
+                     "$ref": "#/components/schemas/Key"
+                  }
+               }
+            ],
+            "responses": {
+               "202": {
+                  "description": "OK",
+                  "content": {
+                     "application/json": {
+                        "schema": {
+                           "$ref": "#/components/schemas/Id"
+                        }
+                     }
+                  }
+               },
+               "400": {
+                  "$ref": "#/components/responses/badRequest"
+               },
+               "5XX": {
+                  "$ref": "#/components/responses/internalServerError"
+               }
+            }
+         },
+         "get": {
+            "tags": [
+               "W3Name HTTP API"
+            ],
+            "summary": "Retrieve a record",
+            "description": "Retrieve an IPNS Record.",
+            "operationId": "get-record",
+            "security": [],
+            "parameters": [
+               {
+                  "name": "key",
+                  "in": "path",
+                  "required": true,
+                  "schema": {
+                     "$ref": "#/components/schemas/Key"
+                  }
+               }
+            ],
+            "responses": {
+               "200": {
+                  "description": "OK",
+                  "content": {
+                     "application/json": {
+                        "schema": {
+                           "$ref": "#/components/schemas/Record"
+                        }
+                     }
+                  }
+               },
+               "400": {
+                  "$ref": "#/components/responses/badRequest"
+               },
+               "5XX": {
+                  "$ref": "#/components/responses/internalServerError"
+               }
+            }
+         }
+      }
+   },
+   "components": {
+      "schemas": {
+         "Id": {
+            "type": "object",
+            "properties": {
+               "id": {
+                  "$ref": "#/components/schemas/Key"
+               }
+            },
+            "description": "The published IPNS Key that will rersolve the record once it has persisted to th DHT."
```

Comment:
"resolve _to_ the record"?

Reply:
yep - makes more sense like that, will update